src/libraries/System.Private.CoreLib/src/System/Char.cs

```diff
@@ -200,29 +201,37 @@ bool ISpanFormattable.TryFormat(Span<char> destination, out int charsWritten, Re
 
         public static char Parse(string s)
         {
-            if (s == null)
-            {
-                ThrowHelper.ThrowArgumentNullException(ExceptionArgument.s);
-            }
+            ArgumentNullException.ThrowIfNull(s);
+            return Parse(s.AsSpan());
+        }
 
+        internal static char Parse(ReadOnlySpan<char> s)
+        {
             if (s.Length != 1)
             {
-                throw new FormatException(SR.Format_NeedSingleChar);
+                ThrowHelper.ThrowFormatException_NeedSingleChar();
             }
             return s[0];
         }
 
         public static bool TryParse([NotNullWhen(true)] string? s, out char result)
         {
-            result = '\0';
-            if (s == null)
+            if (s is null)
             {
+                result = '\0';
                 return false;
             }
+            return TryParse(s.AsSpan(), out result);
+        }
+
+        internal static bool TryParse(ReadOnlySpan<char> s, out char result)
+        {
             if (s.Length != 1)
             {
+                result = '\0';
                 return false;
             }
+
```

Comment:
Uninteresting nit: just curious about the blank line here but not above the final return in the previous method.

Reply:
Just a habit because not having the newline when there are two statements "bugs" me 😄 

For whatever reason, my brain says a "attaching" a single statement, like a return, immediately following a closing curly is fine. But if there are two or more statements, it just "flows incorrectly" and it needs the newline.